.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish on tag
+on:
+  release:
+    types: [published]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+jobs:
+  publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: ⬇️ Set up code
+        uses: actions/checkout@v3
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.7
```

Comment:
**question**: should this match the arcgis pro python version?

Reply:
I was wondering—doesn't this just define the python environment used to build the package (ie, run `setup.py`), not the actual python environment used/expected by the script?

Reply:
Yes, it's the python version used to build and package the module. I would assume there is a less chance of any weirdness if they are the same.